modules/hcp-ec2-client/templates/nginx.conf

```diff
@@ -2,46 +2,6 @@ events {}
 http {
     include /etc/nginx/conf.d/*.conf;
 
-    server {
-        server_name localhost;
-        listen 80 default_server;
-
-        proxy_http_version 1.1;
-
-        proxy_set_header Upgrade $http_upgrade;
-        proxy_set_header Connection "Upgrade";
-        proxy_set_header Host $host;
-        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
-
-        proxy_temp_file_write_size 64k;
-        proxy_connect_timeout 10080s;
-        proxy_send_timeout 10080;
-        proxy_read_timeout 10080;
-        proxy_buffer_size 64k;
-        proxy_buffers 16 32k;
-        proxy_busy_buffers_size 64k;
-        proxy_request_buffering off;
-        proxy_buffering off;
-
-        location / {
-            proxy_pass http://127.0.0.1:3000;
-        }
-
-        location /static {
-            proxy_cache_valid 60m;
-            proxy_pass http://127.0.0.1:3000;
-        }
-
-        location /api {
-            proxy_pass http://127.0.0.1:7070;
-        }
-
-        error_page   500 502 503 504  /50x.html;
-        location = /50x.html {
-            root   /usr/share/nginx/html;
-        }
-    }
-
     server {
         server_name localhost;
         listen 8081;
```

Comment:
this can be a later-thing, but after you remove the above, can we remove `nginx` entirely from here + `setup.sh`?: https://github.com/hashicorp/terraform-aws-hcp-consul/blob/main/modules/hcp-ec2-client/templates/setup.sh#L48-L53

AFAICT, after this fix to move `nginx` into Nomad, we're left with nginx only forwarding from port 8081 to 4646, and it seems fine to just expose port 4646 over in `nomad_url`: https://github.com/hashicorp/terraform-aws-hcp-consul/blob/main/examples/hcp-ec2-demo/output.tf#L15